cl/lib/scorched_utils.py

```diff
@@ -72,3 +72,74 @@ def update(self, extra_options):
 
     def options(self):
         return self.option_dict
+
+
+class MoreLikeThisHighlightsSolrSearch(ExtraSolrSearch):
+    """
+    By default Solr MoreLikeThis queries do not support highlighting. Thus, we need to produce the highlights in Python.
+
+    A MoreLikeThis search with highlight fields that are taken directly from search results
+    """
+
+    # Limit length of text field
+    text_max_length = 500
+
+    # highlight fields (from search_utils)
+    highlight_fields = [
```

Comment:
I don't like how the highlight_fields are hard coded here. Could they be passed as an init function or something?

Reply:
Agree. Then, `search_utils.add_highlighting` must be refactored as well. Maybe move everything to the models or settings?

Reply:
Yep, that sounds good to me.

Reply:
Err, I mean, moving these to settings sounds good to me.